src/libraries/Common/src/System/Net/Http/aspnetcore/Quic/Implementations/MsQuic/Internal/MsQuicSession.cs

```diff
@@ -22,7 +22,7 @@ public IntPtr ConnectionOpen(QuicClientConnectionOptions options)
         {
             if (!_opened)
             {
-                OpenSession(options.ClientAuthenticationOptions.ApplicationProtocols[0].Protocol.ToArray(),
+                OpenSession(options.ClientAuthenticationOptions!.ApplicationProtocols![0].Protocol.ToArray(),
```

Comment:
These group of classes seems only for testing and written in very nullabe ignorant way, so i intended to suppress warnings instead of trying to validate